docs/tox.ini

```diff
@@ -32,3 +32,10 @@ commands_pre =
 commands =
     sphinx-autobuild . _build/html
 
+[testenv:commands]
+description = Generate reference docs for CLI commands
+deps =
+    ruff==0.5.4
```

Comment:
It seems odd to me that the `commands` item also runs the Ruff formatter. Can we have a separate tox command for that? (But maybe we can add that in a separate PR and have it format all the *.py files?)